docs/REST API/rest-api.md

```diff
@@ -4228,20 +4230,21 @@ This endpoint returns the results for a specific custom MDM command.
 
 This endpoint returns the list of custom MDM commands that have been executed.
 
-`GET /api/v1/fleet/mdm/apple/commands`
+`GET /api/v1/fleet/mdm/commands`
 
 #### Parameters
 
 | Name                      | Type    | In    | Description                                                               |
 | ------------------------- | ------  | ----- | ------------------------------------------------------------------------- |
+| platform                  | string  | query | Filter MDM commands by the hosts' platform they ran on. 
 | page                      | integer | query | Page number of the results to fetch.                                      |
 | per_page                  | integer | query | Results per page.                                                         |
 | order_key                 | string  | query | What to order results by. Can be any field listed in the `results` array example below. |
 | order_direction           | string  | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
 
 #### Example
 
-`GET /api/v1/fleet/mdm/apple/commands?per_page=5`
+`GET /api/v1/fleet/mdm/commands?per_page=5?platform=darwin`
```

Comment:
Do we use `darwin` or `macos` for other `platform` query params? I feel like I've seen both. Could be misremembering. Should probably make a call here and stick to it.

Reply:
Seems that we use `darwin`. I've searched through API docs and there's no `"platform": "macos"`